src/main/java/org/opentripplanner/api/model/Place.java

```diff
@@ -91,6 +92,16 @@
      */
     public String bikeShareId;
 
+    /**
+     * Type of board or alight (to distinguish flexible transit modes).
+     */
+    public BoardAlightType boardAlightType;
```

Comment:
I added some documentation for this. The intention was that this would be "BoardAlightType.default" in the vast majority of cases, and if the boarding or alighting is nonstandard in some way, it would be another value. Right now, the only nonstandard cases are for GTFS-Flex, but that won't necessarily always be the case. I added documentation to that effect. But another approach could be to make this flex specific and leave null otherwise. Any drawbacks/benefits to switching approaches?

Reply:
I still find that the name "BoardAlightType" is very general, and the documentation here and on the enum is basically tautological: it's a "type of board or alight". There is still not much information about what this field means or what it is used for.